packages/utilities/src/serialization/annotation-remover.ts

```diff
@@ -0,0 +1,24 @@
+import {Annotation, Language, MemoisingSymbolTable, SerializationChunk} from "@lionweb/core"
+
+/**
+ * Removes all annotation-related instance data from the given {@link SerializationChunk}, i.e.:
+ *  * all nodes that are instances of an {@link Annotation},
+ *  * all annotation on nodes.
+ * @param serializationChunk - The {@link SerializationChunk}
+ * @param languages - An array of {@link Language languages} to determine what nodes are instances of annotations
+ */
+export const withoutAnnotations = (serializationChunk: SerializationChunk, languages: Language[]) => {
+    const symbolTable = new MemoisingSymbolTable(languages)
+    const {serializationFormatVersion, languages: dependentLanguages, nodes} = serializationChunk
+    return {
+        serializationFormatVersion,
+        languages: dependentLanguages,
+        nodes: nodes
+            .filter((node) => !(symbolTable.entityMatching(node.classifier) instanceof Annotation)) // removes instances of annotations
```

Comment:
If the classifier cannot be found the node is not recognised as an Annotation node and the node will be kept in the resulting chunk. This chunk then is incorrect as such an annotation node will be in there without a parent.

An algorithm that does not have this problem would be to remove all annotation id's from each node's annotation array, keep the removed id's and them remove all the nodes with these id's.
The result is now guaranteed to be an (at least as) valid LionWeb chunk. 

And you don't even need the 'languages' parameter.

Reply:
OK, will do that.